### DISCUSSION TITLE
Where is github actions runner view? I don't see it on the settings page (Personal Pro)

### DISCUSSION THREAD

**Discussion by tddschn** [community/community#31329] [Actions]
Where is [github actions runner view](https://github.blog/2022-02-23-new-way-understand-github-hosted-runner-capacity/)?  
I don't see it on the settings page and I'm on the github.com Personal Pro plan.

**Comment by Simran-B** [comment]
What settings page did you check?

This [roadmap card](https://github.com/orgs/github/projects/4247/views/1?filterQuery=actions+runner) links to the changelog:
https://github.blog/changelog/2021-09-20-github-actions-experience-refresh-for-the-management-of-self-hosted-runners/

According to the changelog, it should be "in the Actions settings page of your repository or organization".
